app/src/main/java/ch/epfl/skysync/MainActivity.kt

```diff
@@ -36,6 +41,16 @@ class MainActivity : ComponentActivity() {
           this.onSignInResult(res)
         }
 
-    setContent { SkySyncTheme { LoginScreen(signInLauncher = signInLauncher, user = user.value) } }
+    setContent {
+      SkySyncTheme {
+        val navController = rememberNavController()
+        NavHost(navController = navController, startDestination = Route.LOGIN) {
```

Comment:
Just to be sure will it always take us to login screen when launching app or if we are already logged in we will  end up at home screen directly?

Reply:
I've modified this after this commit to take us to home screen if we already are logged in